emoji/src/main/java/com/vanniktech/emoji/Utils.java

```diff
@@ -48,12 +50,19 @@ static int dpToPx(@NonNull final Context context, final float dp) {
     return (int) (dp * context.getResources().getDisplayMetrics().density);
   }
 
-  static int screenHeight(@NonNull final Activity context) {
-    final Point size = new Point();
+  static boolean shouldOverrideRegularCondition(@NonNull final Activity context, EditText editText) {
 
-    context.getWindowManager().getDefaultDisplay().getSize(size);
+    if ((editText.getImeOptions() & EditorInfo.IME_FLAG_NO_EXTRACT_UI) == 0) {
+      Configuration configuration = context.getResources().getConfiguration();
+      return (configuration.orientation == Configuration.ORIENTATION_LANDSCAPE);
+    }
 
-    return size.y;
+    return false;
+  }
+
+  static int screenHeight(@NonNull final Activity context) {
+    Configuration configuration = context.getResources().getConfiguration();
+    return (dpToPx(context, configuration.screenHeightDp));
```

Comment:
nit: the first `(` & `)` are unnecessary

Reply:
They might be, but I like them. But given it's your guidelines, I'll remove them.